features/user/managing_customers/browsing_customers.feature

```diff
@@ -0,0 +1,17 @@
+@managing_customers
+Feature: Browsing customers
+    In order to see all customers in the store
+    As an Administrator
+    I want to browse customers
+
+    Background:
+        Given the store has customer "f.baggins@example.com"
+        And the store has customer "mr.banana@example.com"
+        And the store has customer "l.skywalker@example.com"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Browsing currencies in store
+        When I want to see all customers in store
+        Then I should see 4 customers in the list
```

Comment:
Count customers accounts without administrator (in Page)? Nothing special, but it make easier reading this scenario.